src/openforms/formio/service.py

```diff
@@ -26,21 +26,17 @@ def get_dynamic_configuration(
     configuration = handle_custom_types(
         configuration, request=request, submission=submission
     )
-    configuration = insert_variables(
-        configuration, submission=submission, request=request
-    )
+    configuration = insert_variables(configuration, submission=submission)
```

Comment:
self-reminder: this conflicts with the changes in #1869, rebase that feature branch after this PR is merged